hw/ip/otbn/data/otbn.hjson

```diff
@@ -134,17 +134,34 @@
   registers: [
     { name: "CMD"
       desc: "command register",
-      swaccess: "r0w1c",
+      swaccess: "wo",
       hwaccess: "hro",
       hwext: "true",
       hwqe:  "true",
       fields: [
-        { bits: "0"
-          name: "start"
+        { bits: "1:0"
```

Comment:
It's a pity that we can't expose "wipe both" here, since that's probably going to be a common thing to want to do. Maybe we could change to something like: "1 => EXECUTE", "2 => WIPE_DMEM", "4 => WIPE_IMEM", "6 => WIPE_BOTH"? (But not allow things like 3, 5 or 7, which wouldn't really make sense).

Reply:
> It's a pity that we can't expose "wipe both" here, since that's probably going to be a common thing to want to do. 

Wipe both isn't that frequent. An IMEM wipe is only needed during a context switch to an untrusted application, while a DMEM wipe is done after every operation (after reading out the result).

So the trade-off is:
- Support a "wipe both" functionality in HW for the benefit of improving the performance of the situation where we do a context switch to an mutually untrusted app.
- Or require SW to issue two back-to-back commands when wanting to wipe both IMEM and DMEM, effectively costing the SW one interrupt latency plus one bus write, plus a couple lines of code. 

Since we already have a fair amount of complexity in the command, busy and error interactions in OTBN, I went for the route here that makes things simpler on the HW.

Reply:
Ok, fine with me.